src/Mvc/test/Mvc.IntegrationTests/ValidationIntegrationTests.cs

```diff
@@ -2013,8 +2013,10 @@ private class Order12
         public IList<Order12> RelatedOrders { get; set; }
     }
 
-    [Fact]
-    public async Task Validation_ListOfType_NoValidatorOnParameter()
+    [Theory]
+    [InlineData("?[0]=1&[1]=2", true)]
+    [InlineData("?[0]=1&[1]=null", false)]
```

Comment:
Could we make these separate tests please? These are essentially two different scenarios masquerading as inputs to the test.